### PR TITLE
Feature/select multiple nodes in one go for custom seeds

### PR DESCRIPTION
**Pull request by MHove89**
Added support for selecting multiple nodes. 

We are adding an array targets in the dialog view, instead of return a single target. 
Support for this scenario has been set up in both the seed view and select node view.

### CODE REVIEW

**Review by jesperweber — COMMENTED** [review]
Looking good and thanks for the fix of the spelling error :-)

I think we are missing the include descendants functionality?

File: src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/selectNode.controller.js [hunk 1]
```diff
@@ -44,13 +51,19 @@
...
+                    name: "Everything",
+                    icon: "icon-item-arrangement",
+                    includeDescendants: true
+                });
 
             } else {
-                $scope.model.target.includeDescendents = vm.includeDescendents;
+                for (let i = 0; i < $scope.model.targets.length; i++) {
+                    
+                    $scope.model.targets[i].includeDescendants = vm.includeDescendants;
```
Comment (MHove89) [inline src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/selectNode.controller.js:65]: Hi @jesperweber 

Include descendants should be set here. I am making sure that it set on all selected nodes for that specific session of the dialog. 

Is it not working for you?

Comment (jesperweber) [reply src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/selectNode.controller.js:65]: Hmm it does seems to be working now, maybe it was just a caching issues on my side?

Well it looks nice and is working :-)

**Review by jesperweber — APPROVED** [review]
Look good.

File: src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/selectNode.controller.js (hunk 1 above)
Comment (MHove89) [reply src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/selectNode.controller.js:65]: It was probably some browser caching :)

**Comment by jesperweber** [comment]
@MHove89 Will you merge the PR if it's ready or should I?

**Pull request merged**